src/colorprint.cc

```diff
@@ -168,7 +168,8 @@ bool IsColorTerminal() {
       "xterm",         "xterm-color",     "xterm-256color",
       "screen",        "screen-256color", "tmux",
       "tmux-256color", "rxvt-unicode",    "rxvt-unicode-256color",
-      "linux",         "cygwin",
+      "linux",         "cygwin",          "xterm-kitty",
+      "alacritty",
```

Comment:
Reference for `alacritty` supporting color?

Reply:
Under `CSI m`: https://github.com/alacritty/alacritty/blob/edf4df66c9afcb9593eaeaaf34d5c80c66925adc/docs/escape_support.md#csi-control-sequence-introducer---esc-
terminfo: https://github.com/alacritty/alacritty/blob/edf4df66c9afcb9593eaeaaf34d5c80c66925adc/extra/alacritty.info